SECP-service/src/main/java/com/visucius/secp/models/Message.java

```diff
@@ -0,0 +1,66 @@
+package com.visucius.secp.models;
+
+import javax.persistence.*;
+import java.util.Date;
+
+
+@Entity
+@Table(name = "message")
+public class Message {
+
+    @Id
+    @GeneratedValue
+    @Column(name = "id", unique = true, nullable = false)
+    private int id;
+
+    @Column(name = "timestamp")
+    @Temporal(TemporalType.TIMESTAMP)
+    private Date timestamp;
+
+    @Column(name = "body", nullable = false)
+    @Lob
+    private String body;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    public Message(Date timestamp, String body, User user) {
+        this.timestamp = new Date(timestamp.getTime());
```

Comment:
I am guessing this is getting the current time and setting the timestamp. If that's the case, why are you putting a parameter in the constructor of the message.